docs/lima.md

```diff
@@ -0,0 +1,47 @@
+# Using stargz-snapshotter on Lima
+
+This document describes how to get started with stargz-snapshotter on [Lima](https://github.com/lima-vm/lima).
```

Comment:
Needs a brief explanation of Lima, otherwise readers won't get the point of this document.